src/overlays/Overlay.tsx

```diff
@@ -76,14 +79,17 @@ export const Overlay = (props: OverlayProps): any => {
     elForPortal &&
     createPortal(
       <CSSTransition
+        nodeRef={nodeRef}
```

Comment:
These changes to the `Overlay` component are necessary for the latest version of `react-transition-group`.